tests/integration/test_interaction.py

```diff
@@ -193,3 +218,78 @@ class CustomHtmlDataInteractionTest(InteractionTestFixture, BaseIntegrationTest)
 
     def _get_scenario_xml(self):
         return self._get_custom_scenario_xml("integration/data/test_html_data.json")
+
+
+class MultipleBlocksDataInteraction(InteractionTestBase, BaseIntegrationTest):
+    PAGE_TITLE = 'Drag and Drop v2 Multiple Blocks'
+    PAGE_ID = 'drag_and_drop_v2_multi'
+
+    BLOCK1_DATA_FILE = "integration/data/test_data.json"
+    BLOCK2_DATA_FILE = "integration/data/test_data_other.json"
+
+    item_maps = {
+        'block1': {
+            0: ItemDefinition(0, 'Zone 1', "Yes 1", "No 1"),
+            1: ItemDefinition(1, 'Zone 2', "Yes 2", "No 2", "102"),
+            2: ItemDefinition(2, None, "", "No Zone for this")
+        },
+        'block2': {
+            10: ItemDefinition(10, 'Zone 51', "Correct 1", "Incorrect 1"),
+            20: ItemDefinition(20, 'Zone 52', "Correct 2", "Incorrect 2", "102"),
+            30: ItemDefinition(30, None, "", "No Zone for this")
+        },
+    }
+
+    all_zones = {
+        'block1': ['Zone 1', 'Zone 2'],
+        'block2': ['Zone 51', 'Zone 52']
+    }
+
+    feedback = {
+        'block1': {"intro": "Other Intro Feed", "final": "Other Final Feed"},
+        'block2': {"intro": "Some Intro Feed", "final": "Some Final Feed"},
```

Comment:
@e-kolpakov Could you switch the strings for initial and final feedback between the `.json` files? I found it slightly confusing that the data from `test_data.json` has "*Other* Intro Feed"/"*Other* Final Feed" while the data coming from `test_data_other.json` has "*Some* Intro Feed"/"*Some* Final Feed".